src/components/Timeline/src/styles/step-label.css

```diff
@@ -0,0 +1,36 @@
+.denhaag-timeline__step-label {
+  display: flex;
+}
+
+.denhaag-timeline__step-label__text {
+  text-overflow: ellipsis;
+  overflow: hidden;
+
+  display: flex;
+  align-items: center;
+
+  margin-block-start: 0;
+  margin-block-end: 0;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+}
+
+.denhaag-timeline__step-label__text:hover {
+  overflow: visible;
+  white-space: normal;
+  height: auto;
+}
```

Comment:
These look like they should be in an MUI overrides file.

Reply:
These are not overwrites from material, but for the standard browser styling.